IFC4x3/Properties/m/MitigatedRiskLikelihood_2mh3cZyunB0ONymM2LqXn/Documentation.md

```diff
@@ -1 +0,0 @@
-Identifies the likelihood of the hazard given the planned mitigation.
```

Comment:
Accidental deletion please revert, this element is still referenced in the Pset_Risk will cause an error in IfcDoc.

Reply:
Refer above about duplicate.